### ISSUE TITLE
Replace "Admin" terminology in permissions

### ISSUE THREAD

**Issue by Chief-Engineer** [space-wizards/space-station-14#30414]
Continuation of #28451 

- Rename the `ADMIN` permission flag. The only suggestion that I've been able to think of is `PRIVILEGEDUSER`
- Rename `IsAdmin` to something like `IsPrivilegedUser` and create an obsoleted `IsAdmin` function that just redirects to the renamed one to avoid breaking forks. Ensure upstream uses are renamed to avoid creating tons of warnings.
- Modify the dead chat "ADMIN(username)" display. Some options:
  - Replace "ADMIN" with "STAFF"
  - Make it a new flag control it

The purposes of these changes are:
- To avoid contributors mistakenly using `IsAdmin` or the `ADMIN` flag when intending to give a permission to game admins. My understanding is that admin was not intended to refer to game admins
- To reduce in-game confusion between staff with elevated permissions and game admins

**Comment by chavonadelal** [comment]
I don't think this is a good idea, because in all forks, any player who has received any rights is an administrator. It doesn't matter if he's a game admin or just a staff member. This is equivalent. And rather, this change will create confusion in terminology.

**Comment by chavonadelal** [comment]
> I'm not sure what you mean.
> 
> > in all forks, any player who has received any rights is an administrator.
> 
> There is no way that wizden is the only server of all SS14 servers where there are people who have some in-game permissions but are not game admins. I think I've been told that there are some russian servers which give people ahelp permissions to act as mentors.
> 
> > And rather, this change will create confusion in terminology. The exception does not work that every admin is a staff, but not every staff is an admin. Think about forks.
> 
> I don't understand what you're trying to say here. All game admins are staff, but not all staff are game admins. Calling all staff admins is inaccurate if any one of those people is not an admin, but calling all admins staff is still accurately labeling them

All members of the administration are called admins. Regardless of whether they have admin access to the game. That's the way it's done.. Because it is literally called the "admin staff". We do not take cases of illiterate management when admin rights are distributed to everyone just like that. Yes, there are helpers who have nothing but the rights to write to ahelp. But they are admins anyway.

**Comment by chavonadelal** [comment]
> In my experience, most players and contributors assume that when someone says "admin", they mean "game admin". I have never heard a non-game admin refer to themselves as an admin or administrator in the context of SS14, and I would consider it misleading if anyone who was not a game admin, project manager, or host did so. That's the primary reason why I think the change needs to be made.
> 
> Responding specifically to what you wrote: I cannot imagine how roles like "helper" fall under the umbrella of "administration". A person who only answers questions but has no ability to do anything else is not in an administrative role. A person who only has access to vv and debugging tools for the purposes of debugging is not in administrative role. Arguably, not even someone who has access to a wide set of admin permissions for the sole purpose of running events on the servers is an administrator
> 
> https://dictionary.cambridge.org/dictionary/english/administrator

This is the main difference of the Russian-speaking segment. For most, any member of the administration is an admin. And if you have any rights in the game, then you are definitely an admin. Admin is not only about having any rights, it is about the attitude towards the server administration.

**Comment by chavonadelal** [comment]
> well the flags in the code are written in English and aren't player facing, so presumably changing the word used as the flag won't negatively impact forks. The deachat should be a localization string so russian forks should be able to make it say whatever they want regardless of what we do here. While I don't know that they do, I assume that russian forks already translate it to say admin in russian rather than in english

What about adding localization to flags in that case?

**Comment by chavonadelal** [comment]
by the way i don't understand your logic in light of the fact that in my pull request you said that every admin has all the rights to solve problems and now you say that not every player with rights is an admin

**Comment by LordEclipse** [comment]
> by the way i don't understand your logic in light of the fact that in my pull request you said that every admin has all the rights to solve problems and now you say that not every player with rights is an admin

At least in Wizden, In Wizden ADMIN is used to EXCLUSIVELY refer to people that are Game Admins that punish problem players according to policy and rules, and sometimes host events, only when they are a full admin and not a trial admin. An ADMIN is somebody that has Volunteered by putting forth an Application to become an Administrator to deal with player facing administration issues. 

Maintainers are CODERS who do not have any expectation nor have they put in an application to deal with player facing administration issues. 

SOME Project Managers are Admins, SOME Project Managers are Maintainers, SOME Project Managers are both.

**Comment by chavonadelal** [comment]
> > by the way i don't understand your logic in light of the fact that in my pull request you said that every admin has all the rights to solve problems and now you say that not every player with rights is an admin
> 
> At least in Wizden, ADMIN is used to EXCLUSIVELY refer to people that are Game Admins that punish problem players according to policy and rules, and sometimes host events, only when they are a full admin and not a trial admin. An ADMIN is somebody that has Volunteered by putting forth an Application to become an Administrator to deal with player facing administration issues. 
> 
> Maintainers are CODERS who do not have any expectation nor have they put in an application to deal with player facing administration issues. 
> 
> SOME Project Managers are Admins, SOME Project Managers are Maintainers, SOME Project Managers are both.

In the ru segment, any member of the administration is considered an admin even if he has no rights in the game. And I did not understand what you are getting at if in my pull request to add a position in ahelp, they told me the opposite, that it is not necessary and everyone with rights can help. And now it turns out that not all of you are admins and should help directly in ahelp.

**Comment by chavonadelal** [comment]
> > What about adding localization to flags in that case?
> 
> None of the flags have any localization. Can you explain what burden it would cause for forks if the flag was named something like `PRIVILEGEDUSER` instead of `ADMIN` in the code?
> 
> > by the way i don't understand your logic in light of the fact that in my pull request you said that every admin has all the rights to solve problems and now you say that not every player with rights is an admin
> 
> Can you provide a link to where I said this please? Also those things aren't in conflict, even if admins have all rights it doesn't mean that everyone with any rights is an admin.
> 
> > they told me the opposite, that it is not necessary and everyone with rights can help. And now it turns out that not all of you are admins and should help directly in ahelp.
> 
> On Wizard's Den, everyone with the ahelp permission has equal ability to warn a player or tell them that something is against the rules. There are things that game admins aren't allowed to do, but in the context of an ahelp, none of that is something that is both affected by the game admin's rank and that we expect or want players to enforce.
> 
> This is the same situation with every fork that I'm personally familiar with.
> 
> The only instance that I'm aware of where there is any difference in what someone can do in ahelp and a server would want to communicate rank for that purpose is if someone is in a mentor type position so only has ahelp permissions. In that situation, my understanding is that ahelp already changes the color of their name to communicate this

Many forks have their own sponsorship (privilege) systems. Therefore, such names of flags can be misleading. I've just reread the comments and I'm sorry, it wasn't you who wrote it, but another admin. In any case, our dispute lies in the cultural differences between the understanding of the word "admin" and those who relate to this word.
